common/djangoapps/third_party_auth/migrations/0002_samlproviderconfig_country.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.14 on 2020-07-21 19:08
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('third_party_auth', '0001_squashed_0026_auto_20200401_1932'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='samlproviderconfig',
+            name='country',
+            field=models.CharField(blank=True, help_text=('String representation of the mapping between user`s IDP and edx`s country field.',), max_length=255),
```

Comment:
being nitpicky on whether we need max_length 255, but not a major issue. Are country names going to be ful names or abbreviations?

Reply:
guess it's okay but good to check, try looking into maybe the EnterpriseCustomer model it has a similar field I bet

Reply:
I thought this field was not actually providing a country name but rather providing the name of the field as it exists in their IDP. Ie our "country" maps to their "locale" or something.

Reply:
either way, probably needlessly long.

Reply:
255 is most likely too long here. Talked with @taliaedX and I'm confident that this isn't going to be an abbreviation, but unsure if it's the country name, name of the IDP's country field or the actual urn supplied by the IDP. I've pinged Rob for more clarity and will report what he has to say. If it does end up being the urn (my guess, the longest of the choices), what do you guys think the max length should be?

Reply:
Yeah now that I revisited that ticket 
```
{
"urn": "https://federation-sts.customer.com/schemas/claims/1/countrycode",
"name": "country"
}
```

it does appear that it's the urn, would be good to confirm. If so, then 255 may be needed but not too sure either https://docs.microsoft.com/en-us/linkedin/shared/api-guide/concepts/urns#:~:text=URN%20values%20returned%20from%20LinkedIn's,of%20255%20characters%20in%20length.

Reply:
Our other URN-expecting fields seem to use a 128 char size.

Reply:
Yeah, looking at the other fields in the model, I agree with Talia. It seems like all URN's are 128 and default values are 255, so I think my choice would be to make it 128.